comments-bundle/src/Resources/contao/classes/Comments.php

```diff
@@ -558,6 +559,9 @@ public static function addCommentsSubscription(CommentsModel $objComment)
 
 		$time = time();
 
+		// Make sure the request URL does not contain non-ascii characters (see #4708)
+		$request = (string) (new Uri(Environment::get('request')));
```

Comment:
This should be added here as well, shouldn‘t it?

https://github.com/contao/contao/blob/f247bc966e59bea997b5c06683c20a95b780b729/core-bundle/src/Resources/contao/library/Contao/Search.php#L61-L69

Reply:
Nyholm's implementation differs slightly - for BC we should probably keep it the way it is now. I personally wouldn't mind changing it though. @ausi wdyt?

Reply:
I’d change the the Search.php to `Nyholm\Psr7\Uri` too, as it handles all cases, and all of them correctly.